jobs/postgres/spec

```diff
@@ -5,26 +5,40 @@ packages:
   - postgres-9.4
 
 templates:
-  bin/postgres_ctl:         bin/postgres_ctl
+  bin/ctl:         bin/ctl
   bin/postgres_start.sh:    bin/postgres_start.sh
   config/postgresql.conf:   config/postgresql.conf
   config/pg_hba.conf:       config/pg_hba.conf
 
+provides:
+- name: db
+  type: sql
+  properties:
+  - port
+  - db_scheme
+  - roles
+  - databases
+
 properties:
-  databases.port:
+  port:
     description: Port to listen on
+    default: 5432
+
+  db_scheme:
+    description: Link property to declare it as postgres://
+    default: postgres
```

Comment:
Yeah this one I think is explicitly for links - to expose the type of the db for consumers. Not really for overriding; just for passing out to the consumers of the link.